dracut-module-setup.sh

```diff
@@ -349,6 +360,50 @@ kdump_setup_ifname() {
     echo "$_ifname"
 }
 
+_clone_nmconnection() {
+    local _clone_output _name _unique_id
+
+    _unique_id=$1
+    _name=$(nmcli --get-values connection.id connection show "$_unique_id")
+    if _clone_output=$(nmcli connection clone --temporary "$_unique_id" "$_name"); then
```

Comment:
> this bash scripting is as nice as it gets.
> 
> Interestingly, a major reason why NetworkManager exists is to provide an API for applications. Despite that, it is still very cumbersome to do such things in a shell script. The future `nmcli --offlne` command might make that slightly easier. But the main difficulty is that the script has to look at the configuration and autonomously understand how that configuration can be modified to still make sense. That seems fundamentally non-trivial.

That's exactly how I feel when I use current nmcli to edit the .nmconnection file. And `nmcli --offlinine` is exactly what I want:) Maybe we can assume the users know what they are going to do. Btw, if there are any documentation about the pitfalls or some restraints I should respect when editing .nmconnection, please let me know. 

> 
> anyway, this approach lgtm, the patch too.

Thanks for reviewing the patches, I really appropriate it!

Reply:
Hi Thomas,

Thank you for approving the changes. If the latest version also looks good to you, can I add  `Reviewed-by: Thomas Haller <thaller@redhat.com>` to each commit message? 

For your reference, here's the meaning of "Reviewed-by" tag quoting from the [Linux Kernel Documentation](https://www.kernel.org/doc/html/latest/process/submitting-patches.html#reviewer-s-statement-of-oversight), 
> By offering my Reviewed-by: tag, I state that:
>
> I have carried out a technical review of this patch to evaluate its appropriateness and readiness for inclusion into the mainline kernel (kexec-tools).
>
> Any problems, concerns, or questions relating to the patch have been communicated back to the submitter. I am satisfied with the submitter’s response to my comments.
>
> While there may be things that could be improved with this submission, I believe that it is, at this time, (1) a worthwhile modification to the kernel (kexec-tools), and (2) free of known issues which would argue against its inclusion.
>
> While I have reviewed the patch and believe it to be sound, I do not (unless explicitly stated elsewhere) make any warranties or guarantees that it will achieve its stated purpose or function properly in any given situation.

Reply:
The branch looks good to me. Excellent work!

Reply:
Reviewed-by: Thomas Haller <thaller(at)redhat.com>

Reply:
Great, thanks!